scripts/github-actions/update-browser-versions.js

```diff
@@ -67,22 +72,28 @@ const getVersions = async ({ core }) => {
 
 const checkNeedForBranchUpdate = ({ core, latestStableVersion, latestBetaVersion }) => {
   // file path is relative to repo root
-  const branchBrowserVersions = JSON.parse(fs.readFileSync('./browser-versions.json'))
-  const hasNewerStableVersion = branchBrowserVersions['chrome:stable'] !== latestStableVersion
-  const hasNewerBetaVersion = branchBrowserVersions['chrome:beta'] !== latestBetaVersion
+  const doc = yaml.parseDocument(fs.readFileSync('./.circleci/workflows.yml', 'utf8'))
+
+  const currentChromeStable = doc.contents.items.find((item) => item.key.value === 'chrome-stable-version').value.value
+  const currentChromeBeta = doc.contents.items.find((item) => item.key.value === 'chrome-beta-version').value.value
```

Comment:
Might be worthwhile to create some utility functions to get the workflow yaml and current chrome versions.

Reply:
I was somewhat hesitant to dry this up because we are dealing with object references and like the idea of function purity, but I should be able to at least factor out the keys